api/python/provisioner/config.py

```diff
@@ -502,28 +501,12 @@ class ReleaseInfo(Enum):
 NON_CRITICALLY_FAILED = {"critical": False, "failed": True}
 
 
-class SWUpgradeRepos(Enum):
-    """List of supported SW Upgrade Repositories."""
-
-    OS = "os"  # yum repo
-    third_party = "3rd_party"  # yum repo
-    cortx = "cortx_iso"  # yum repo
-    python = "python_deps"  # python index
+class InstallType(Enum):
+    """List of installation types."""
 
+    base = "base"
```

Comment:
If these are to be treated as ENUM, please define the variables in CAPS.

Reply:
Thank you! I updated that